templates/includes/page_metadata.html

```diff
@@ -0,0 +1,16 @@
+<meta name="description" content="{{ page_description }}" />
+<meta name="keywords" content="{{ page_keywords }}" />
+<title>{{ site_title }} : {{ page_title }}</title>
+
+<meta property="og:type" content="website" />
+<meta property="og:site_name" content="{{ site_title }}" />
+<meta property="og:title" content="{{ site_title }} : {{ page_title }}" />
+<meta property="og:description" content="{{ page_description }}" />
+<meta property="og:url" content="{{ page_url }}" />
+<meta property="og:image" content="{{ base_url }}{{ static_url }}img/logo_squares.png" />
+
+<meta name="twitter: card" content="summary" />
+<meta name="twitter:title" content="{{ site_title }} : {{ page_title }}" />
+<meta name="twitter:description" content="{{ page_description }}" />
+<meta name="twitter:image" content="{{ base_url }}{{ static_url }}img/logo_squares.png" />
+<meta name="twitter:url" content="{{ page_url }}" />
```

Comment:
```suggestion
<meta name="twitter:card" content="summary" />
```
Remove a space in `twitter: card` and also redundant twitter cards that are already defined as generic Open Graph.